public/javascripts/SVLabel/src/SVLabel/keyboard/Keyboard.js

```diff
@@ -274,13 +286,16 @@ function Keyboard (svl, canvas, contextMenu, googleMap, ribbon, zoomControl) {
                             keyCode: e.keyCode
                         });
                         break;
+                    case 16:
+                        lastShiftKeyUpTimestamp = e.timeStamp;
+                        break;
                     case 90:
                         if (contextMenu.isOpen()){
                             contextMenu.hide();
                             svl.tracker.push("KeyboardShortcut_CloseContextMenu");
                         }
                         // "z" for zoom. By default, it will zoom in. If "shift" is down, it will zoom out.
```

Comment:
Could you mention the 100ms buffer here?

Reply:
added the shift comment, and the 100ms comment here